### DISCUSSION TITLE
Quick tags working outside the fast reblog

### DISCUSSION THREAD

**Discussion by snorlana** [AprilSylph/XKit-Rewritten#385] [Help Desk]
Idk if this is supposed to work and atm isn't for some reason but when I click to reblog the post instead of using the fast reblog I cannot use the quick tags.
But either way it'd be useful to be able to use them on the pop up reblog and the post creating area. (sorry if I'm getting the names and terminologies wrong in here xD)

**Comment by AprilSylph** [comment]
Quick Tags in XKit Rewritten only affects the beta post editor.

**Reply by snorlana** [reply]
ah, gotcha. thank you
